.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+	"configurations": [
+		{
+			"name": "Build for Debug",
+			"type": "node-terminal",
+			"request": "launch",
+			"command": "npm run debug"
+		},
+		{
+			"name": "Build for Production",
+			"type": "node-terminal",
+			"request": "launch",
+			"command": "npm run production"
+		},
+		{
+			"name": "Install devDependencies",
```

Comment:
I think `Install NPM Dependencies` sounds better.

Reply:
Alright, Ill make a quick commit to change it